app/templates/components/print-course.hbs

```diff
@@ -113,6 +113,7 @@
                   <th class='text-center'>{{t 'general.type'}}</th>
                   <th class='text-center'>{{t 'general.required'}}</th>
                   <th class='text-left'>{{t 'general.notes'}}</th>
+                  <th class='text-left'>{{t 'general.citation'}}</th>
```

Comment:
I believe there are very few learning materials that are citations.  @saschaben is it worth spending the screen real-estate on a column for this or should it be squeezed into the title column?